src/cow-react/modules/limitOrders/hooks/useApplyLimitRate.ts

```diff
@@ -11,25 +11,22 @@ export function useApplyLimitRate() {
   const { activeRate } = useAtomValue(limitRateAtom)
 
   return useCallback(
-    (value: string | null, field: Field): string | null | undefined => {
+    (value: string | null, field: Field): Fraction | null => {
       if (!value || !Number(value) || !activeRate || activeRate.equalTo(0)) {
         return null
       }
 
-      let output: Fraction | null = null
       const parsedValue = toFraction(value)
 
-      // If the field is INPUT we will MULTIPLY passed value and rate
```

Comment:
i love you kill this comments :) 

Comments are very valuable, but this ones were adding 0 value

Reply:
I'm stick to https://en.wikipedia.org/wiki/Self-documenting_code